domain/media/src/test/kotlin/com/giraffe/media/movies/usecase/ClearMovieCacheWithOutRecentViewedUseCaseTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.giraffe.media.movies.usecase
+
+import com.giraffe.media.movies.repository.MoviesRepository
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+
+class ClearMovieCacheWithOutRecentViewedUseCaseTest {
+
+    private lateinit var repository: MoviesRepository
+
+    @BeforeEach
+    fun setUp() {
+        repository = mockk(relaxed = true)
+    }
+
+
+    @Test
+    fun `should call repository clear method`() = runTest {
```

Comment:
I think this name is better :
```suggestion
    fun `should call clear method in repository`() = runTest {
```